src/locales/en/common.json

```diff
@@ -336,6 +336,7 @@
   "New blocks": "New blocks",
   "New bookmark": "New bookmark",
   "New transactions": "New transactions",
+  "New wallet address": "New wallet address",
```

Comment:
This seems weird this text was added by i18n even though there are no changes to the JSX in this PR

Reply:
so how changed it they didn't addet to the common js there is transaltion for it